stable/redis/templates/metrics-svc.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.metrics.enabled }}
```

Comment:
If there is no service, Prometheus can't scrape metrics from the redis-exporter. Therefore, I think that the service should always enabled when the metrics are enabled.

Reply:
This is not true, you can still scrape the metrics from within the cluster. The service is needed only if you want to scrape the metrics from outside of the cluster, no? Unless, I'm missing something.

Reply:
why not just add this 9121 port to the existing redis Service?

Reply:
There are two ways doing this: Add metrics-service as @7CHANHO is doing, or add redis-exporter as a sidecar container of the redis-master pod. IMHO converting redis-metrics deployment to be a redis-master sidecar would be better because we don't need another service, besides, service redis-master has already added 9121 port for metrics scraping.